### ISSUE TITLE
content addition: forcats

### ISSUE THREAD

**Issue by zolli22** [RhoBott/data_at_reed#51]
I know in my own experience (especially with graphing) and from drop-in questions, "I want these to be in a specific/different order than R/ggplot2 is putting them in" is a somewhat common issue and adding content on how to do that could be good!

main thoughts:
- link to cheat sheet
- what is a factor? what is a level?
- how do I make something a factor?
- how do I change the levels of a factor?
- TL;DR :: how do I get R to read this thing I have in the order I want it to be read?

**Comment by joshyam-k** [comment]
just dropping a note that I started working on this inside of `presentation-visualization` and can pick it up again on Friday, but if someone has free time before then feel free to add to it as well!

**Comment by joshyam-k** [comment]
If others are okay with what's been added I think this issue can be closed! We now have a section with links to `forcats` help and such inside of "advanced wrangling topics" as well as a whole .rmd on using `forcats` functions to reorder your ggplots!

**Comment by zolli22** [comment]
I think everything we mentioned is covered! nice work everyone!
